### ISSUE TITLE
Better differentiate packages for grpc-dotnet from deprecated grpc

### ISSUE THREAD

**Issue by 1R053** [grpc/grpc-dotnet#1314]
Can we use a separate icon for grpc-dotnet related packages when publishing to NuGet?
It would be great, if the packages that are related to grpc-dotnet are better distinguishable from the grpc packages. This may reduce a lot of confusion, questions and issues from users, that somehow incorrectly mix the two code lines together.

**Comment by JamesNK** [comment]
Currently, the gRPC NuGet packages use the old gRPC logo. We could update grpc-dotnet packages to use the new logo...

I suspect what we'll do is mark Grpc.Core as deprecated in a years time - https://docs.microsoft.com/en-us/nuget/nuget-org/deprecate-packages

**Comment by JamesNK** [comment]
@jtattermusch

**Comment by jtattermusch** [comment]
Yes, once Grpc.Core is deprecated (see https://grpc.io/blog/grpc-csharp-future/), we will probably mark the Grpc.Core nugets as deprecated.  At this point we can only do things like add a note in the nugets package description, since we cannot modify the names of existing nugets.

(but at his point the nuget package description doesn't really seem that unclear to me: https://www.nuget.org/packages/Grpc.Core/2.38.0. On the other hand, there's always room for improvement).
